API/src/app.js

```diff
@@ -0,0 +1,32 @@
+import express from 'express';
+import bodyParser from 'body-parser';
+
+import orders from './routes/ordersRoute';
+
+const app = express();
+
+app.use(bodyParser.json());
+app.use(bodyParser.urlencoded({ extended: true }));
+
+app.use('/api/v1', orders);
+
+app.use((request, response, next) => {
+  const error = new Error('unknown url path');
+  error.status = 404;
+  return next(error);
+});
+
+// eslint-disable-next-line no-unused-vars
+app.use((error, request, response, next) => {
+  const status = error.status || 400;
+  response.status(status);
+  return response.json({ status, message: error.message });
+});
+
+const PORT = parseInt(process.env.PORT, 10) || 5000;
+
+if (require.main === module) {
```

Comment:
I am curious about this check. Please explain.

Reply:
@sudhons. You might not need to do that. I just wanted to understand what you'd done. If you find a better way to handle this though. 
I am concerned that in the test environment, what you are checking for might not really matter.